src/storage/mod.rs

```diff
@@ -1501,6 +1501,18 @@ mod tests {
         })
     }
 
+    fn expect_value_callback<T: PartialEq + Debug + Send + 'static>(
+        done: Sender<i32>,
+        id: i32,
+        value: T,
+    ) -> Callback<T> {
+        Box::new(move |x: Result<T>| {
+            assert!(x.is_ok());
```

Comment:
do you need to check ok here?

Reply:
maybe we can just remove this line since the unwrap in the next line is enough.